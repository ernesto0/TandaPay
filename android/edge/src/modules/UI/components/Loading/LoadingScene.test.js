// @flow

/* globals describe it expect */
/* eslint-disable flowtype/require-valid-file-annotation */

import React from 'react'
import ShallowRenderer from 'react-test-renderer/shallow'

import { LoadingScene } from './LoadingScene.ui.js'

describe('LoadingScene', () => {
  it('should render', () => {
    const renderer = new ShallowRenderer()

    const props = {}
    const actual = renderer.render(<LoadingScene {...props} />)

    expect(actual).toMatchSnapshot()
  })
})
