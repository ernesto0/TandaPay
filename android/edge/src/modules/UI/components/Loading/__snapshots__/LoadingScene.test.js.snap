// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingScene should render 1`] = `
<SafeAreaViewComponent
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Component
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Gradient
      style={
        Object {
          "height": 44,
          "width": "100%",
        }
      }
    />
    <Component
      style={
        Object {
          "alignItems": "center",
          "flex": 1,
          "justifyContent": "center",
        }
      }
    >
      <ActivityIndicator
        animating={true}
        color="#999999"
        hidesWhenStopped={true}
        size="large"
      />
    </Component>
  </Component>
</SafeAreaViewComponent>
`;
